assignments/callbacks.js

```diff
@@ -27,23 +27,54 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
+getLength(items, function(first) {
+  console.log(first)
+});
+
+
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[3]);
 }
 
+last(items, function(last) {
+  console.log(last)
+});
+
+
+
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb;
```

Comment:
You'd want to add the result of the sum as an argument to the callback function here, so you could just do:
```js
return cb(x + y);
```